client/state/account-recovery/reset/reducer.js

```diff
@@ -17,20 +17,44 @@ import {
 	ACCOUNT_RECOVERY_RESET_REQUEST_ERROR,
 	ACCOUNT_RECOVERY_RESET_UPDATE_USER_DATA,
 	ACCOUNT_RECOVERY_RESET_PICK_METHOD,
+	ACCOUNT_RECOVERY_RESET_SET_VALIDATION_KEY,
+	ACCOUNT_RECOVERY_RESET_VALIDATE_REQUEST,
+	ACCOUNT_RECOVERY_RESET_VALIDATE_REQUEST_SUCCESS,
+	ACCOUNT_RECOVERY_RESET_VALIDATE_REQUEST_ERROR,
 } from 'state/action-types';
 
+const createRequestingReducer = ( onActions, offActions ) => createReducer( false, {
```

Comment:
I've never seen this pattern but it looks neat 😄.  I'm all for helper functions but I'm not sure I see that its simplifying anything here. The number of lines increased and the readability just about stayed the same. I'd be onboard with it if this were a helper function along side `createReducer` that was used across the board for boolean reducers, something like `createBooleanReducer`. But even adding that would just add more API like functionality to learn/be aware of.